### PR TITLE
some config file and other changes

### PR DESCRIPTION
**Pull request by alanhamlett**
* use ~/.wakatime.cfg as settings file
* remove quotes and unused info from user-agent string
* no longer need to import wakatime plugin for parseConfigFile function

### CODE REVIEW

File: WakaTime.py [hunk 1]
```diff
@@ -7,11 +7,15 @@
 import sys
 import time
 import threading
+import traceback
 from urllib import request
 from zipfile import ZipFile
-from configparser import ConfigParser
 from subprocess import Popen, STDOUT, PIPE
 from queue import Queue, Empty
+try:
```
Comment (allista) [inline WakaTime.py]: Not sure this is necessary, as Blender bundles its custom python-3.5.

File: WakaTime.py [hunk 2]
```diff
@@ -37,23 +41,15 @@
 PLUGIN_DIR = os.path.dirname(os.path.realpath(__file__))
 API_CLIENT_URL = 'https://github.com/wakatime/wakatime/archive/master.zip'
 API_CLIENT = os.path.join(PLUGIN_DIR, 'wakatime-master', 'wakatime', 'cli.py')
-SETTINGS_FILE = os.path.join(PLUGIN_DIR, 'wakatime.ini')
-SETTINGS = None
-settings = None
+SETTINGS_FILE = os.path.join(os.path.expanduser('~'), '.wakatime.cfg')
```
Comment (allista) [inline WakaTime.py]: Is there a reason to use this particular config file (e.g. all wakatime plugins should share it)?

Comment (alanhamlett) [inline WakaTime.py]: Correct, all wakatime plugins share it so if you update your api key you don't have to change it for every editor.

Comment (alanhamlett) [inline WakaTime.py]: Only the Sublime Text plugin uses it's own config file, all others share the same `~/.wakatime.cfg` file.

**Comment by allista** [comment]
Understood. I'll be away for several hours. Then I'll make appropriate changes to use the correct config file. Also, Sublime's plugin attempts to use yet another config location on Windows; shouldn't I do the same here? That's was the primary reason why I chose to use plugin directory to store the config: so it would work on Windows as well.

**Comment by alanhamlett** [comment]
The Sublime plugin looks for the api key in `~/.wakatime.cfg` and uses that as the default when prompting the user, but it uses the `WakaTime.sublime-settings` file from then on:

https://github.com/wakatime/sublime-wakatime/blob/master/WakaTime.py#L185

It uses the same file locations on Windows, Linux, and OS X because `~` is replaced with the user's home folder on the respective OS.

The only reason it uses the `WakaTime.sublime-settings` is because Sublime users are used to that convention and they look there for plugin settings. Since Blender doesn't have a well known convention like Sublime, we should use the common config file in the user's home folder.

**Comment by allista** [comment]
I will close this PR as I have manually applied the crucial change -- the use of the .wakatime.cfg file; but left alone the code restyling and refactoring. Instead I've found some minor bugs and figured out how to show the API Key dialog automatically on plugin start, or if the key is incorrect (I interpret the error 104 as the indication of that). I also updated the README accordingly.

**Pull request closed without merging**